java/com/facebook/yoga/YogaNode.java

```diff
@@ -310,6 +310,18 @@ public void setOverflow(YogaOverflow overflow) {
     jni_YGNodeStyleSetOverflow(mNativePointer, overflow.intValue());
   }
 
+  private native int jni_YGNodeStyleGetDisplay(long nativePointer);
+  @Override
+  public YogaDisplay getDisplay() {
+    return YogaDisplay.values()[jni_YGNodeStyleGetDisplay(mNativePointer)];
```

Comment:
took the changes from GetOverflow and just renamed the name. If you push a commit I'll leave this for now.